lib/shore/client/tokens/merchant_account.rb

```diff
@@ -0,0 +1,111 @@
+# frozen_string_literal: true
+module Shore
+  module Client
+    module Tokens
+      class MerchantAccount # :nodoc:
+        require_relative 'merchant_role'
```

Comment:
Move the requires to the top of the file. That's the ruby convention, I think. Right?